docs/appendices/release-notes/5.2.3.rst

```diff
@@ -0,0 +1,65 @@
+.. _version_5.2.3:
+
+=============
+Version 5.2.3
+=============
+
+Released on 2023-02-24.
```

Comment:
Why is it `24` here and not `23` ?

Reply:
Just thought 23 end of day and 24 are interchangeable. Fixing this. Thank you !